core/editor/editor.php

```diff
@@ -881,6 +727,10 @@ public function __construct() {
 		add_action( 'admin_action_elementor', [ $this, 'init' ] );
 		add_action( 'template_redirect', [ $this, 'redirect_to_new_url' ] );
 
+		// Register editor v2 experiment
+		$this->register_editor_v2_experiment();
```

Comment:
shouldn't it run on some hook?

Reply:
tried with the hook and this is not working because the hook trigger before the register